features/logging/4.x/elasticsearch.feature

```diff
@@ -0,0 +1,69 @@
+@clusterlogging
+Feature: Elasticsearch related tests
+
+  # @author qitang@redhat.com
+  # case_id OCP-21487
+  @admin
+  @destructive
+  @commonlogging
+  Scenario: Elasticsearch Prometheus metrics can be accessed.
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-logging" project
+    And I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | _prometheus/metrics |
+      | op           | GET                 |
+    Then the step should succeed
+    And the output should contain:
+      | es_cluster_nodes_number                   |
+      | es_cluster_shards_active_percent          |
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22050
+  @admin
+  @destructive
+  Scenario: Elasticsearch using dynamic volumes
+    Given I switch to cluster admin pseudo user
+    And default storageclass is stored in the :default_sc clipboard
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    Given I use the "openshift-logging" project
+    And I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/example_unmanaged.yaml |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    And I run the :delete client command with:
+      | object_type | pvc               |
+      | all         | true              |
+      | n           | openshift-logging |
+    Then the step should succeed
```

Comment:
if no PVCs exist, will command fail?

Reply:
I checked in my computer, when there is no pvc exist, the output is `No resources found`, and the command exit code is `0`. How does the cucushift determine whether a command is successfully executed?

Reply:
I checked this using cucushift, the command won't fail if there is no pvc.
```    And I run the :delete client command with:              # features/step_definitions/cli.rb:13
      [01:19:03] INFO> Shell Commands: oc delete pvc --all --config=/home/qitang/workdir/wlc-trust-142-qitang/ocp4_admin.kubeconfig -n openshift-logging
      No resources found
      
      [01:19:05] INFO> Exit Status: 0
      | object_type | pvc               |
      | all         | true              |
      | n           | openshift-logging |
    Then the step should succeed                            # features/step_definitions/common.rb:4
```